src/config/dynamic-theme-fixes.config

```
@@ -5624,6 +5624,36 @@ CSS
 
 ================================
 
+digi.sets.fi/makortrep/ma-kort-repetition-1/
+digi.sets.fi/ma8kort/
+digi.sets.fi/ma7kort/
+digi.sets.fi/ma6kort/
+digi.sets.fi/ma5kort/
+digi.sets.fi/ma4kort/
+digi.sets.fi/ma3kort/
+digi.sets.fi/ma2kort/
+digi.sets.fi/ma1kort/
+digi.sets.fi/ma13lang/
+digi.sets.fi/ma12lang/
+digi.sets.fi/ma11lang/
+digi.sets.fi/ma10lang/
+digi.sets.fi/ma9lang/
+digi.sets.fi/ma8lang/
+digi.sets.fi/ma7lang/
+digi.sets.fi/ma6lang/
+digi.sets.fi/ma5lang/
+digi.sets.fi/ma4lang/
+digi.sets.fi/ma3lang/
+digi.sets.fi/ma2lang/
+digi.sets.fi/ma1lang/
+
+CSS
+.bi {
+    filter: invert(0.936);
+}
```

I think this would better fit:
```suggestion
INVERT
.bi 
```

---

I tried a lot to get this to work and I thought the same way, but for some reason it just doesn't work. The exact amount of invertion on the original technique also looks better as it blends in with the background and doesn't leave half of the book another hue of black than the rest.